captum/optim/_core/objectives.py

```diff
@@ -143,10 +142,8 @@ def optimize(
                 (-1 * loss_value.mean()).backward()

```

Comment:
Suggestion: I think that we may want to remove the `*-1` here, and potentially the `.mean()` as well in favour of the changes that we previously dicussed.

Reply:
I'm not sure if we want to make the applicable objective outputs negative, or have positive objectives output negative values.

Reply:
Perhaps, we can have a loss summarization function that could be provided by user as an argument to optimize method. The default `loss_summarize_fn` will have `-1 * loss_value.mean()` logic. `loss_summarize_fn` will take loss_value as an input.

Reply:
For the loss objectives, their scalar outputs should be like so I think (some may already have the right output, but double check with Lucent / Lucid just encase):

1. LayerActivation: mean()
2. ChannelActivation: mean()
3. NeuronActivation: mean()
4. DeepDream: mean()
5. TotalVariation: sum()
6. L1: sum()
7. L2: None
8. Diversity: None
9. ActivationInterpolation: None
10. Alignment: None
11. Direction: mean()
12. DirectionNeuron: mean()
13. TensorDirection: mean()
14. ActivationWeights: mean()

I think that we also need to double check if the outputs should be negative or positive for at least a few of the objectives